superset/assets/javascripts/explorev2/index.jsx

```diff
@@ -14,6 +14,8 @@ import ExploreViewContainer from './components/ExploreViewContainer';
 import { exploreReducer } from './reducers/exploreReducer';
 import { appSetup } from '../common';
 
+require('./main.css');
```

Comment:
[optional] `import './main.css';` works too, in that previous large PR I got rid of most of the `require` statements